index.html

```diff
@@ -53,13 +53,16 @@ <h1>CODEVID-19</h1>
       </p>
       <ul class="actions">
         <li>
-          <a href="join-the-fight" class="button">Join the Fight</a>
+          <a href="{{ '/join-the-fight' | relative_url }}" class="button primary">Join the Fight</a>
         </li>
         <li>
           <a href="#become-a-sponsor" class="button scrolly"
             >Become a Sponsor</a
           >
         </li>
+        <li>
+          <a href="{{ '/blog' | relative_url }}" class="button">Read Our Blog</a>
```

Comment:
👍

Reply:
Same question I have about the necessity of `relative_url`.  tl;dr: it looks useful in `_layouts/`-level HTML, but not here.

Reply:
The only other time it's useful is when you're serving it from a not root url. I often use it as a rule, but you're right that `relative_url` is not needed if a standard relative link (i.e. `blog.html` or `./blog.html`) would work.

Reply:
I'll leave them in until I've become a bit more familiar with Jekyll templates.